internal/repository/rsl.go

```diff
@@ -187,3 +187,22 @@ func (r *Repository) isDuplicateEntry(refName string, targetID plumbing.Hash) (b
 
 	return latestUnskippedEntry.TargetID == targetID, nil
 }
+
+func LogRSLEntries(repo *Repository) ([]*rsl.ReferenceEntry, map[plumbing.Hash][]*rsl.AnnotationEntry, error) {
```

Comment:
This function name sounds like you want to log RSL entries. I'd rename to something like `GetRSLEntryLog` or so.

Reply:
Also docstring.

Reply:
Fixed